tests/CMakeLists.txt

```diff
@@ -73,12 +73,12 @@ add_subdirectory(win7)
 set(DEBUG_BUILD FALSE)
 set(HAS_DEBUG_INFO FALSE)
 
-if (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+#if (${CMAKE_BUILD_TYPE} STREQUAL "Debug")

```

Comment:
I can only assume that you were testing something?

Reply:
ah yes, sorry : ) I actually wasn't able to get the repo building anymore using the instructions in the README... CMAKE_BUILD_TYPE seems to evaluate to empty string even though I call init_all / init.cmd with the right params. Perhaps something is out of sync in the readme?

Reply:
Interesting. That sounds like it _might_ be related to using MSBuild as the generator? If that's the case, we may need to disable the sanitizers under those circumstances. I'd expect the `init_all` script to work, though as that forces Ninja as the generator

Reply:
Filed #318